src/helpers.rs

```diff
@@ -0,0 +1,45 @@
+extern crate yaml_rust;
+
+use std::fs::File;
+use std::io;
+
+use std::io::prelude::*;
+use std::process; 
+
+use self::yaml_rust::{YamlLoader};
+
+#[derive(Debug)]
+pub struct Config {
+    // dirs content is String so Config owns its string data
+    pub dirs: Vec<String>,
```

Comment:
That comment is really only relevant because we are n00bs. Better documentation would be to describe what this `dirs` value represents:
`"The root directory (or directories) in which all project source code repos live"`